### PR TITLE
Retrieve Cart Items by user_id & temp remove decode token

### PR DESCRIPTION
**Pull request by ocarvente**
route completed. removed middleware for the time being.

### CODE REVIEW

File: server/controllers/index.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 const model = require('../models');
 
 module.exports = {
-  getCartByUserId: (req, res) => {
-    res.end();
+  getCartByUserId: async(req, res) => {
+    const{user_id} = req.params;
```
Comment (pnilan) [inline server/controllers/index.js]: You'll want to get the userId from req.headers.userId since this requires authentication.

**Review by pnilan — CHANGES_REQUESTED** [review]

**Review by ocarvente — COMMENTED** [review]
How does this look?

File: server/controllers/index.js [hunk 2]
```diff
@@ -7,12 +7,20 @@ module.exports = {
...
-      res.status(404).send();
+      res.status(404).send(error);
     }
   },
 
-  createTransaction: (req, res) => {
-    res.end();
+  createTransaction: async (req, res) => {
+
+    const{user_id} = req.headers;
```
Comment (ocarvente) [inline server/controllers/index.js:16]: This

File: server/controllers/index.js (hunk 1 above)
Comment (ocarvente) [reply server/controllers/index.js]: So i believe i actually made this update in the commit 'added middleware back and changed routes with user id params. ' You must have seen an older version

**Review by pnilan — APPROVED** [review]
👍

**Pull request merged**